08-Frac_calc/FracCalc.java

```diff
@@ -24,14 +24,14 @@ public static String produceAnswer(String input) {
         String operand2Numerator = findNumerator(operand2); //find fraction 2 numerator
         String operand2Denominator = findDenominator(operand2); //find fraction 2 denominator
 
-        int op1w = Integer.parseInt(operand1Whole);
+        int op1w = Integer.parseInt(operand1Whole); //convert string into int
```

Comment:
Naming is one of the hardest parts of CS - what are these variable names? I'm having trouble figuring out what they stand for... Could you make them a little clearer?